src/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs

```diff
@@ -66,6 +68,19 @@ internal AcquireTokenInteractiveParameterBuilder WithCurrentSynchronizationConte
             return this;
         }
 
+        internal AcquireTokenInteractiveParameterBuilder WithParentActivityOrWindowFunc(Func<object> parentActivityOrWindowFunc)
+        {
+#if RUNTIME || NETSTANDARD_BUILDTIME
+
+            if (parentActivityOrWindowFunc != null)
+            {
+                WithParentActivityOrWindow(parentActivityOrWindowFunc());
```

Comment:
What happens if both `WithParentActivityOrWindow` and `WithParentActivityOrWindowFunc` are used? We should also capture the desired behavior in an unit test

Reply:
There is a test for that.  See TestAcquireTokenInteractive_ParentWindow_WithCallbackFuncAndAcquireTokenBuilder_Async in this PR